Monika After Story/game/definitions.rpy

```diff
@@ -5500,13 +5500,25 @@ init -1 python:
 
         RETURNS: True if a proccess in proc_list is running, False otherwise
         """
-        running_procs = get_procs()
-        if len(running_procs) == 0:
-            return False
-
-        for proc in proc_list:
-            if proc in running_procs:
-                return True
+        if renpy.linux:
```

Comment:
We can probably put the windows check in this position, then have the following stuff in an `else` to fallback for Mac/Linux then